aries-site/src/pages/index.js

```diff
@@ -1,69 +1,109 @@
+  Featured,
+  Highlights,
+  Quote,
+  Video,
+  WhatIs,
+} from '../components/home';
+import { Layout } from '../layouts';
+import { getPageDetails, useDarkMode } from '../utils';
 
 const title = 'Home';
 const pageDetails = getPageDetails(title);
-const cards = getCards();
-const featured = [
-  'Designer Guidance',
-  'Forms',
-  'Header',
-  'Human Centered',
-  'Developer Guidance',
-  'Button',
-  'Navigation',
-  'Icons',
-  'Search',
-  'Typography',
-];
 
-const featuredCards = [];
-const cardOrder = [];
+const calcPad = size => {
+  const val = size !== 'small' ? 'xlarge' : 'large';
+  return val;
+};
+
+const Intro = ({ children }) => {
+  const size = useContext(ResponsiveContext);
+  const darkMode = useDarkMode();
+  const image = `/home${darkMode.value ? '-dark' : ''}.svg`;
 
-cards.forEach(card => {
-  if (featured.includes(card.name)) featuredCards.push(card);
-  else cardOrder.push(card);
-});
+  return (
+    <Stack guidingChild="last">
+      <Box
+        align="start"
+        justify="between"
+        pad={{ horizontal: calcPad(size) }}
+        direction="row"
+      >
+        {size !== 'small' && <Box width="small" />}
+        <Box>
+          <Image src={image} fit="contain" />
+        </Box>
+      </Box>
+      <Box height={{ min: 'medium' }} justify="center">
+        <Grid
+          gap="large"
+          columns={size === 'small' ? ['auto'] : ['3/4', 'auto']}
+        >
+          <Card background="none" elevation="none">
+            {children}
+          </Card>
+          {size !== 'small' && (
+            <Card background="none" elevation="none" height="small" />
+          )}
+        </Grid>
+      </Box>
+    </Stack>
+  );
+};
 
-featuredCards.sort(
-  (a, b) => featured.indexOf(a.name) - featured.indexOf(b.name),
-);
+Intro.propTypes = {
+  children: PropTypes.node,
+};
 
-cardOrder.splice(0, 0, ...featuredCards);
+const Index = () => {
+  const size = useContext(ResponsiveContext);
 
-const Index = () => (
-  <Layout
-    backgroundImage={{
-      src: { dark: '/cards-dark.svg', light: '/cards.svg' },
-      alt: 'HPE Design System',
-      margin: { top: '50px', left: '-150px' },
-      style: { transform: 'scale(1.4)', transformOrigin: 'top left' },
-      small: {
-        margin: { left: '-75px', top: '-75px' },
-      },
-      useGrid: true,
-    }}
-    title={title}
-    isLanding
-  >
-    <Meta title={title} description={pageDetails.seoDescription} />
-    <Box gap="large">
-      <PageIntro>
-        <Box justify="center" fill>
-          <Heading margin="none">HPE Design System</Heading>
-          <Paragraph size="large" fill>
-            The Design System was created to empower designers, developers, and
-            others to contribute to making great experiences for the customer.
-          </Paragraph>
-        </Box>
-      </PageIntro>
-      <CardGrid cards={cardOrder} />
-    </Box>
-  </Layout>
-);
+  return (
+    <Layout title={title} isLanding pad={{}} width={{}}>
```

Comment:
I think my impression was that we still want the content within the page to have a max-width of "xxlarge" like the rest of the pages but that the background colors themselves would be able to take full width. However, it would be worth confirming. @vavalos5 @halocline

Reply:
Done